feature_layers/display-subtype-feature-layer/src/main/java/com/esri/samples/display_subtype_feature_layer/DisplaySubtypeFeatureLayerController.java

```diff
@@ -67,8 +66,9 @@ public void initialize() {
       ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_STREETS_NIGHT);
       mapView.setMap(map);
       // display the current map scale
-      mapView.addMapScaleChangedListener(mapScaleChangedEvent ->
-        currentMapScaleLabel.setText("Current map scale: 1:" + Math.round(mapView.getMapScale())));
+      mapView.mapScaleProperty().addListener((observable, oldValue, newValue) -> {
```

Comment:
Based on the new knowledge gained in this review https://github.com/Esri/arcgis-runtime-samples-java/pull/755#pullrequestreview-1165176984

you could perhaps explore something like this:

```
      currentMapScaleLabel.textProperty().bind(Bindings.createStringBinding(() -> {
        return "Current map scale: 1:" + Math.round(mapView.mapScaleProperty().get());
      }, mapView.mapScaleProperty()));
```